hr_attendance_sheet/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Pavlov Media
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "HR Attendance Sheet",
+    "version": "12.0.1.0.0",
+    "category": "Human Resources",
+    "summary": "Group attendances into attendance sheets.",
+    "website": "https://github.com/OCA/hr",
+    "author": "Odoo S.A.," "Odoo Community Association (OCA)",
+    "license": "LGPL-3",
```

Comment:
```suggestion
    "license": "AGPL-3",
```

This module is AGPL or LGPL?

Reply:
It's AGPL, I fixed all the references to LGPL and change it. Thanks.